### PR TITLE
feat/used @verida/verifiable-credentials package  to create vcs

### PR DESCRIPTION
**Pull request by cmcWebCode40**
- Updated Lib to send Vc using the `@verida/verifiable-credentials` package

#31

### CODE REVIEW

**Comment by nick-verida** [comment]
@andy-verida can you please review this change

**Comment by nick-verida** [comment]
Doesn't build

```
➜  backend git:(feature/31-use-verifiable-credential-lib) ✗ npm run start

> backend@0.0.1 start /Users/nlothain/dev/verida/Credentials-Management-Platform/backend
> nest start

src/helpers/VeridaHelper.ts:5:29 - error TS2307: Cannot find module '@verida/verifiable-credentials' or its corresponding type declarations.

5 import { Credentials } from "@verida/verifiable-credentials";
                              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Found 1 error(s).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! backend@0.0.1 start: `nest start`
```

**Review by nick-verida — CHANGES_REQUESTED** [review]
Doesn't build

**Comment by cmcWebCode40** [comment]
@nick-verida, 

i have fixed this issue , you can now build the backend .

File: backend/src/helpers/VeridaHelper.ts [hunk 1]
```diff
@@ -226,13 +121,13 @@ export default class VeridaHelper {
...
-    const context = Network.connect({
-      context: {
-        name: VERIDA_APP_NAME,
-      },
+
+
+    const context = await Network.connect({
       client: {
-        environment: VERIDA_ENVIRONMENT,
+        environment: EnvironmentType.TESTNET,
```
Comment (andy-verida) [inline backend/src/helpers/VeridaHelper.ts:128]: Better to use VERIDA_ENVIRONMENT. Why do we need to hardcode this?

**Review by andy-verida — APPROVED** [review]
I just have a question, otherwise, looks good.

File: backend/src/helpers/VeridaHelper.ts (hunk 1 above)
Comment (nick-verida) [reply backend/src/helpers/VeridaHelper.ts:128]: @cmcWebCode40 Fix this and I think I can merge this.

Comment (nick-verida) [reply backend/src/helpers/VeridaHelper.ts:128]: Unfortunately I need to deploy this now. I've raised a new issue for this.

**Review by nick-verida — APPROVED** [review]
I've raised a new issue from Andy's review.

Merging this

**Pull request merged**